src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -1689,11 +1704,25 @@ namespace HVACUnitarySystem {
 			} else {
 			}}
 
-			// IF small loads to meet, just shut down unit
+			// IF small loads to meet or not converging, just shut down unit
 			if ( std::abs( ZoneLoad ) < Small5WLoad ) {
 				ZoneLoad = 0.0;
 				CoolingLoad = false;
 				HeatingLoad = false;
+			} else if( iterationCounter  > 4 ) { // attempt to lock output (air flow) if oscillations are detected
+				if( QToCoolSetPt < 0.0 ) {
+					HeatingLoad = false;
+					CoolingLoad = true;
+					ZoneLoad = QToCoolSetPt;
+				} else if( QToHeatSetPt > 0.0 ) {
+					HeatingLoad = true;
+					CoolingLoad = false;
+					ZoneLoad = QToHeatSetPt;
+				} else {
+					HeatingLoad = false;
+					CoolingLoad = false;
+					ZoneLoad = 0.0;
+				}
```

Comment:
I could pick a value closer to 20, but that would mean the sim would unnecessarily iterate. I've looked at the output of this model many times. If the load is 240 W, the system outputs 240 W every iteration. 

I'd welcome comments.